src/history.h

```diff
@@ -276,6 +278,9 @@ class history_search_t {
     /// Returns the current search result item. asserts if there is no current item.
     const history_item_t &current_item() const;
 
+    /// Returns whether the current item exists, which indicates whether a search had any results.
+    bool current_item_exists() const;
```

Comment:
This was another attempt to check for no matches, but it doesn't seem to be useful because `current_item` is null before `go_to_next_match()` is called, even if there are matches.

Reply:
current_item is only set after the first go_to_next_match()